soccer/gameplay/skills/capture.py

```diff
@@ -98,12 +107,19 @@ def bot_in_front_of_ball(self):
     def find_intercept_point(self):
         return find_robot_intercept_point(self.robot)
 
+    def find_moving_intercept(self):
```

Comment:
Any reason to have a `find_moving_intercept` method instead of just directly calling `find_moving_robot_intercept(self.robot)` instead?